src/Formatter/NegotiatedFormatter.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Equip\Formatter;
+
+use Equip\Exception\FormatterException;
+use Equip\Resolver\ResolverTrait;
+use Equip\Structure\SortedDictionary;
+use Negotiation\Negotiator;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Relay\ResolverInterface;
+
+class NegotiatedFormatter extends SortedDictionary
```

Comment:
It doesn't need to be used that way, since it acts as wrapper.

```php
/** @var NegotiatedFormatter */
return $formatter->apply($request, $response, $content);
```